src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -467,6 +472,61 @@ bool CascadiaSettings::_isPowerShellCoreInstalledInPath(const std::wstring_view
     return false;
 }
 
+// Function Description:
+// - Adds all of the WSL profiles to the provided container.
+// Arguments:
+// - A ref to the profiles container where the WSL profiles are to be added
+// Return Value:
+// - <none>
+void CascadiaSettings::_CreateWslProfiles(std::vector<TerminalApp::Profile>& profileStorage)
+{
+    try
+    {
+        DWORD bytesAvailable = 0;
+        DWORD exitCode = 0;
+        std::wstring command = L"wsl.exe --list";
+        std::wstring wline = L"";
+        std::wstring distName = L"";

```

Comment:
I don't think the explicit initializer is needed.

Reply:
Removed them all.  Cheers,